src/frontend/tsconfig.json

```diff
@@ -12,11 +12,11 @@
     "module": "ESNext",
     "moduleResolution": "Node",
     "resolveJsonModule": true,
-    "isolatedModules": true,
+    "isolatedModules": false,
```

Comment:
Hey Bing, were we getting any specific error/warn that we are removing this default behaviour?

Reply:
Yes, if set to true, when run npm build to generate the bundle file, it will give below error 
` error TS1208: 'landingpage.cy.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.`

Reply:
well, it is complaining that there is no import or export in the file. Can't you fix it by adding an empty import like you did in commands.ts or component.ts.

My whole point is to be consistent. Either remove empty export from the other files OR add it in `landingpage.cy.ts` too and revert isolatedModules to true.

My preference - I'd keep isolatedModules. My reading is very limited to this link below. I am up for a conversation if you have any additional info that I may not be considering. https://www.typescriptlang.org/tsconfig#isolatedModules.

Reply:
change back and add a empty export